lib/bdf/src/BdfTokenStream.php

```diff
@@ -53,16 +53,18 @@ public function advance(): void
      */
     public function parseLine(): string
     {
-        $taken = $this->takeWhile(fn (?string $token) => $token !== "\n");
+        $taken = $this->takeWhile(
+            static fn (string $token): bool => $token !== PHP_EOL
```

Comment:
`PHP_EOL` is platform dependent IIRC. really we should be checking for any type of new line `\n`, `\r`, `\r\n` or whatever, but otherwise happy to keep it with `"\n"`

Reply:
Got it. Let's leave "\n" for now.